cob_bringup/robots/raw3-3.xml

```diff
@@ -63,6 +63,9 @@
 		<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+
+		<include file="$(find rplidar_ros)/launch/rplidar.launch" />
```

Comment:
@ipa-fxm yes, the package can be found here: https://github.com/robopeak/rplidar_ros
I have also added the run dependency as suggested.